### PR TITLE
Adds rainbow jumpsuit

### PR DESCRIPTION
**Pull request by SweptWasTaken**
This is the one from Goonstation.

### CODE REVIEW

**Review by DrSmugleaf — APPROVED** [review]

File: Resources/Textures/Clothing/Uniforms/color.rsi/meta.json [hunk 1]
```diff
@@ -2,8 +2,10 @@
   "version": 1,
   "size": {
     "x": 32,
-    "y": 32
-  },
+    "y": 32,
+      },
+  "license": "CC-BY-SA-3.0/CCA-NC-SA-3.0",
```
Comment (ike709) [inline Resources/Textures/Clothing/Uniforms/color.rsi/meta.json:7]: Please don't put NC icons in the same RSI files as non-NC icons... That'll probably cause problems in the future.

Comment (RemieRichards) [reply Resources/Textures/Clothing/Uniforms/color.rsi/meta.json:7]: it means we can't tell which are which...

also `CCA-` isn't a valid license segment, guess typo'd?

**Comment by SweptWasTaken** [comment]
Mergeable

File: Resources/Textures/Clothing/Uniforms/rainbow.rsi/meta.json [hunk 2]
```diff
@@ -0,0 +1,74 @@
+{
+  "version": 1,
+  "size": {
+    "x": 32,
+    "y": 32,
+      },
+  "license": "CC-3-BY-NC-SA",
```
Comment (DrSmugleaf) [inline Resources/Textures/Clothing/Uniforms/rainbow.rsi/meta.json:7]:
```suggestion
  "license": "CC BY-NC-SA 3.0",
```

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**